src/base/bittorrent/session.cpp

```diff
@@ -2110,7 +2111,9 @@ bool Session::loadTorrent(LoadTorrentParams params)
 {
     lt::add_torrent_params &p = params.ltAddTorrentParams;
 
+    #if LIBTORRENT_VERSION_NUM < 20000
```

Comment:
Yet another try to add libtorrent-2.0 support by removing existing features. Disapproved by me.

Reply:
Please consider #13395 for the first time. Maybe you'll have some new ideas.